### DISCUSSION TITLE
求了，谁能教教我怎么登录啊，更新不了，还总跳到网站上，已经注册了，但我看的视频上明明写了不用网站登录，更新也更不了，谁能教一下谢谢

### DISCUSSION THREAD

**Discussion by zzz20100624** [share121/inter-knot#4929] [委托]
![img-1510239338_1726359622436](https://github.com/user-attachments/assets/09bae6da-bdc1-4a8c-87c0-5d95cf050789)

**Comment by aoki-ensei** [comment]
我也想问

**Comment by hk0852** [comment]
你得在这个网址上面找到更新的内容：https://github.com/share121/inter-knot/releases

**Comment by yansai0v0** [comment]
同问

**Comment by cnNanGua** [comment]
**同问**啊啊啊啊啊啊啊啊啊

**Comment by yunxuan000** [comment]
我的→用户接口

**Comment by frisk666666** [comment]
在我的里把公开接口调成用户接口，然后用你的账号比如github和绳网关联，一般会给你一串数字代码，填写完成就关联上了，准备就绪了重进一下绳网即可显示你的帐号了
